lms/djangoapps/discussion_api/serializers.py

```diff
@@ -191,12 +191,12 @@ class ThreadSerializer(_ContentSerializer):
     closed = serializers.BooleanField(read_only=True)
     following = serializers.SerializerMethodField()
```

Comment:
Should `pinned` and `followed` be a `BooleanField` as well?  While you are in here, it seems like it would be nice to have consistency.